lib/controller/api/ApiAgendaController.class.php

```diff
@@ -81,7 +81,7 @@ public function getAgenda() {
 		$maaltijdAanmeldingen = array_keys(MaaltijdAanmeldingenModel::instance()->getAanmeldingenVoorLid($mids, $_SESSION['_uid']));
 
 		// Sorteren
-		usort($result, array('AgendaModel', 'vergelijkAgendeerbaars'));
+		usort($result, array('CsrDelft\model\agenda\AgendaModel', 'vergelijkAgendeerbaars'));
```

Comment:
Is er iets vergelijkbaars mogelijk voor die `'vergelijkAgendeerbaars'` of moet dat een string blijven?